blackbox/docs/sql/administration/user_defined_functions.txt

```diff
@@ -127,7 +142,10 @@ This would overload our ``multiply`` function with more arguments::
 .. NOTE::
 
    It is considered bad practice to create functions that have the same name
-   as the CrateDB built-in functions.
+   as the CrateDB built-in functions. When no schema name is provided, the
```

Comment:
Even if it's slightly longer i would prefer, to have it more clear. How about this?

... built-in functions. If you call a function without a schema name, CrateDB will look up the builtin functions first and then the user-defined functions in the session schema. Therefore a built-in function with the same name as a user defined function will hide the latter. However, ...